src/theory/bv/bv_solver_bitblast.cpp

```diff
@@ -408,6 +417,37 @@ void BVSolverBitblast::handleEagerAtom(TNode fact, bool assertFact)
   registeredAtoms.clear();
 }
 
+std::vector<Node> BVSolverBitblast::getProofNodes(proof::DratProof dratProof)
+{
+  NodeManager* nm = NodeManager::currentNM();
+  Node cl = nm->mkBoundVar("cl", nm->booleanType());
+  Node del = nm->mkBoundVar("del", nm->booleanType());
+  Node lastFalseResolution = nm->mkConst<bool>(false);
+  prop::SatLiteral zeroLiteral = prop::SatLiteral(0);
+
+  std::vector<Node> args;
+  for (const proof::DratInstruction instruction : dratProof.getInstructions())
+  {
+    if (instruction.d_clause.size() == 0
+        || instruction.d_clause[0] == zeroLiteral)
+    {
+      args.push_back(nm->mkNode(kind::SEXPR, {cl, lastFalseResolution}));
+      break;
+    }
+    std::vector<Node> clauseNodes;
+    clauseNodes.emplace_back(
+        instruction.d_kind == proof::DratInstructionKind::DELETION ? del : cl);
+    for (const prop::SatLiteral literal : instruction.d_clause)
+    {
+      Node fact = d_cnfStream->getNode(literal);
+      Assert(!fact.isNull());
+      clauseNodes.emplace_back(fact);
```

Comment:
```suggestion
      Assert(d_cnfStream->hasLiteral(literal));
      clauseNodes.emplace_back(d_cnfStream->getNode(literal));
```

I think this actually what we want to test here. A saved literal will never be associated with a null node, so it suffices to test that the literal exists in the cnf stream (if it did not then `getNode` would fail anyway).